lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -1,10 +1,10 @@
 <%page args="section_data" expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
-
+<h2 id="header-${section_data['section_key']}">${_("Student Admin")}</h2>
 <div>
   %if section_data['is_small_course']:
   ## Show the gradebook for small courses
-    <h2>${_("Student Gradebook")}</h2>
+    <h3 class="hd hd-2">${_("Student Gradebook")}</h3>
```

Comment:
Same comment here. h3 is looking compressed with the text that follows.
![image](https://cloud.githubusercontent.com/assets/2004678/15298704/991a4220-1b6e-11e6-9fb3-0b45719908db.png)